internal/character/qingque/skill.go

```diff
@@ -0,0 +1,44 @@
+package qingque
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Skill key.Modifier = "qingque-skill"
+)
+
+func init() {
+	modifier.Register(Skill, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+		Stacking:   modifier.ReplaceBySource,
+		MaxCount:   4,
+		Listeners: modifier.Listeners{
+			OnAdd: skillOnAdd,
+		},
```

Comment:
missing listener for this buff to get removed at the end of the turn. In dm, they call `RemoveSelf` in `OnPhase2`